src/site/asciidoc/manual/appenders.adoc

```diff
@@ -1270,12 +1270,12 @@ new `<JMS/>` configuration element.
 [cols=",,,",options="header",]
 |=======================================================================
 |Parameter Name |Type |Default |Description
-|allowdLdapClasses|String|null|A comma separated list of fully qualified class names that may be accessed by LDAP.
-The classes must implement Serializable. Only applies when the JMS Appender By default only Java primative classes
+|allowedLdapClasses|String|null|A comma separated list of fully qualified class names that may be accessed by LDAP.
+The classes must implement Serializable. Only applies when the JMS Appender By default only Java primitive classes
 are allowed.
-|allowdLdapHosts|String|null|A comma separated list of host names or ip addresses that may be accessed by LDAP.
+|allowedLdapHosts|String|null|A comma separated list of host names or ip addresses that may be accessed by LDAP.
 By default only the local host names and ip addresses are allowed.
-|allowdJndiProtocols|String|null|A comma separated list of protocol names that JNDI will allow. By default only java,
+|allowedJndiProtocols|String|null|A comma separated list of protocol names that JNDI will allow. By default only java,
```

Comment:
This developer facing documentation is wrong as these fields don't exist in these misspelled forms.

Reply:
Normally my tooling doesn't merge commits together, but this probably happened because of a merge conflict when I sorted my commits. Sorry about that.